### DISCUSSION TITLE
Not working anymore

### DISCUSSION THREAD

**Discussion by FablabDescartes** [EvilHacker/BoxJoint#13] [General]
Hi,

Love this add-on, it's very useful for CNC or laser cutting. Thanks for making it !
I've used it without any issue a couple of months ago, but now it looks like it does not work anymore. I can only select faces and the options bellow have disappeared. There is just the OK and Cancel buttons.
I use Fusion 360 on Mac, and my version is up to date.

<img width="1440" alt="Capture d’écran 2024-11-17 à 13 32 04" src="https://github.com/user-attachments/assets/90b6ed4e-122e-432e-a11c-2938de47457f">

**Comment by EvilHacker** [comment]
Hmm, that looks completely broken.

I really don't know what it could be. Can you please check some things to help figure out what is wrong.

* Are there any error messages? `View -> Show Text Commands ⌥⌘C` menu to see console logs.
* I wonder if there are some files missing. Did you install from the Autodesk App Store or from github? Can you try to delete and then re-install the Add-In?
* Is this a problem with Box Joints both in existing design and new designs?

**Reply by EvilHacker** [reply]
Could you share this model? It would be helpful for me to bug the problem. Thanks

**Comment by FablabDescartes** [comment]
Hi.
Thanks for your reply. Reinstalling the add-in manually solved the problem !
